extensions/wikia/Wall/WallExternalController.class.php

```diff
@@ -268,11 +290,11 @@ public function deleteMessage() {
 		switch( $this->request->getVal( 'mode' ) ) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add a "case default" clause to this "switch" statement. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS131)